src/components/Facility/DoctorNote.tsx

```
@@ -33,24 +43,41 @@ const DoctorNote = (props: DoctorNoteProps) => {
+          {state.notes.map((note) => {
+            const noteCard = (
               <PatientNoteCard
                 note={note}
                 setReload={setReload}
                 disableEdit={disableEdit}
                 setReplyTo={setReplyTo}
+                mode={mode}
+                allowThreadView={mode === "thread-view"}
+                allowReply={mode !== "thread-view"}
+                setThreadViewNote={setThreadViewNote}
               />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)